debugger/src/main/resources/logback.xml

```diff
@@ -27,7 +27,17 @@
       <pattern>%date{ISO8601} [%thread] %highlight(%-5level) %cyan(%logger{15}) - %msg %n</pattern>
     </encoder>
   </appender>
-  <root level="DEBUG">
+
+  <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+      <file>${LOG_FILE}</file>
+      <append>false</append>
+      <encoder>
+          <pattern>%date{ISO8601} [%thread] %highlight(%-5level) %cyan(%logger{15}) - %msg %n</pattern>
```

Comment:
Concern: we don't want color highlighting in the files, they will just look weird when sent through normal tools like `less`. Or at least that's how it used to be.

Reply:
Gotcha that would explain the weird stuff I was seeing in the file